src/app/api/event/route.js

```diff
@@ -4,6 +4,7 @@ import { withErrorHandler } from '@/lib/errors/errorHandler';
 export const GET = withErrorHandler(async req => {
   const today = new Date();
   const currentYear = today.getFullYear();
+  const currentMonth = parseInt(today.getMonth() + 1, 10);
```

Comment:
@Yivgha this is already number, so there is no reason to parse it

Reply:
changed